File: 8_Kyu_Kata/fake_binary/JavaScript/fakeBinary.js
// 8 kyu - Fake Binary  [ ID: 57eae65a4321032ce000002d  (fake-binary) ]
// URL: https://www.codewars.com/kata/57eae65a4321032ce000002d
// Category: undefined  |  Tags: FUNDAMENTALS | STRINGS | ARRAYS
// *****************************************************************************
function fakeBin(x){
    x = x.replace(/[1-4]/g, "0")
    x = x.replace(/[5-9]/g, "1")
    return x
  
  }

File: 8-kyu-kata/fake-binary/python/fake_binary.py
# 8 kyu - Fake Binary  [ ID: 57eae65a4321032ce000002d  (fake-binary) ]
# URL: https://www.codewars.com/kata/57eae65a4321032ce000002d
# Category: REFERENCE  |  Tags: FUNDAMENTALS | STRINGS | ARRAYS
# ******************************************************************************
def fake_bin(x):
    return "".join(list(map(lambda v: "0" if int(v) < 5 else "1", list(x))))
